### DISCUSSION TITLE
How do I add an array parameter?

### DISCUSSION THREAD

**Discussion by seifaldeen1** [Sketchware-Pro/Sketchware-Pro#764] [Q&A] [answered]
![Screenshot_٢٠٢٣٠٢١٤-١٣٢٠٤٦](https://user-images.githubusercontent.com/114912723/218711451-8ac886a0-6ee7-4a63-bc49-5795c277c4c7.jpg)

**Comment by seifaldeen1** [comment]
How do I add an array parameter?

**Comment by hasrat-ali** [comment, marked as answer]
It's work same as `m.WebView[]`

![Screenshot_2023-02-16-01-04-39-94](https://user-images.githubusercontent.com/42956453/219134305-8e0ce95c-bae7-460f-a638-f8e2751a3ef1.jpg)
